packages/react/src/migrations/update-13-3-0/update-external-emotion-jsx-runtime.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  addProjectConfiguration,
+  readProjectConfiguration,
+} from '@nrwl/devkit';
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+
+import { updateExternalEmotionJsxRuntime } from './update-external-emotion-jsx-runtime';
+
+describe('updateExternalEmotionJsxRuntime', () => {
+  it('should update external for Emotion', async () => {
+    // ARRANGE
+    const tree = createTreeWithEmptyWorkspace(2);
+    addProjectConfiguration(tree, 'components', {
+      root: 'libs/components',
+      targets: {
+        build: {
+          executor: '@nrwl/web:package',
```

Comment:
`@nrwl/web:package` has been renamed to `@nrwl/js:rollup`
https://github.com/nrwl/nx/pull/7904